### PR TITLE
Applied material theme to buttons

### PR DESCRIPTION
**Pull request by clundin55**

### CODE REVIEW

File: yw_website/yw_website/apps/website/static/design/common.css [hunk 1]
```diff
@@ -59,3 +59,3547 @@ body {
 .filler {
     height: .1em;
 }
+@charset "UTF-8";
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/static/design/common.css:68]: if we're copy and pasting css we should say where our source is

Comment (clundin55) [reply yw_website/yw_website/apps/website/static/design/common.css:68]: done

**Review by aniehuser — APPROVED** [review]
@clundin55 thanks for doing that i know its annoying

File: yw_website/yw_website/apps/website/templates/account/base.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     {% load static %}
     {% get_static_prefix as STATIC_PREFIX %}
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="{{ STATIC_PREFIX }}design/detailed_design.css">
+    <link rel="stylesheet" type="text/css" href="{{ STATIC_PREFIX }}design/common.css">
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/templates/account/base.html:8]: oh good catch

**Review by aniehuser — APPROVED** [review]

File: yw_website/yw_website/apps/website/templates/account/base.html (hunk 2 above)
Comment (clundin55) [reply yw_website/yw_website/apps/website/templates/account/base.html:8]: > @clundin55 thanks for doing that i know its annoying

No biggie I wrongly assumed it was already in there

**Pull request merged**